darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -1327,7 +1357,42 @@ def load_model(path: str) -> "TorchForecastingModel":
 
                 from darts.models import RNNModel
 
-                model_loaded = RNNModel.load_model("my_model.pth.tar")
+                model_loaded = RNNModel.load("my_model.pth.tar")
+            ..
+
+        Parameters
+        ----------
+        path
+            Path under which to save the model at its current state. The path should end with '.pth.tar'
```

Comment:
Do we want to keep enforcing that the path ends with `.pth.tar`? The default save path does not seem to satisfy this.

Reply:
I think Dustin had a look at this and found that `.pth.tar` is more of a relic. It looks like `.pt` is now recommended. With the current code I don't think we have to enforce any file ending and just append `.ckpt` to the checkpoint file.

The parameter description needs to be updated though :)

Reply:
OK, what you suggest sounds good 👍